stock_move_source_relocate_dynamic_routing/tests/test_dynamic_destination.py

```diff
@@ -12,9 +14,11 @@ def test_pickship_with_routing_and_relocate(self):
         destination location and one not assigned relocated and then re-routed
         to that same destination location. As both pick moves are going to the
         same destination location, ensure the ship move is not split.
+        As the pick moves share the same characteristics, they are merged back
+        together.
```

Comment:
On which event this is occurring? If the second pick move becomes available then it is merged back in the second?